mungegithub/mungers/release-note-label_test.go

```diff
@@ -74,14 +74,14 @@ func TestReleaseNoteLabel(t *testing.T) {
 		{
 			name:        "LGTM with release-note-label-needed",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel, releaseNoteLabelNeeded}, true),
-			mustHave:    []string{releaseNoteLabelNeeded},
-			mustNotHave: []string{lgtmLabel},
+			mustHave:    []string{doNotMergeLabel, releaseNoteLabelNeeded},
+			mustNotHave: []string{},
 		},
 		{
 			name:        "LGTM only",
 			issue:       github_test.Issue(botName, 1, []string{lgtmLabel}, true),
-			mustHave:    []string{releaseNoteLabelNeeded},
-			mustNotHave: []string{lgtmLabel},
+			mustHave:    []string{doNotMergeLabel, releaseNoteLabelNeeded},
```

Comment:
Can we add lgtmLable to "mustHave" to capture the behavior that we don't remove it when it's applied already?